projects/mybyte/components/navbar.tsx

```diff
@@ -79,9 +79,9 @@ const Navbar = ({ children }: { children: React.ReactNode }) => {
           " overflow-y-auto font-inter"
         }
       >
-        <nav className="bg-white border-gray-200 shadow-md">
+        <nav className="bg-white border-gray-200 shadow-md sticky top-0 z-10">
```

Comment:
this is the only line that was changed in my commit, everything else is a result of rebasing